tests/nightly/nightly-kickoff.sh

```diff
@@ -75,10 +75,13 @@ for i in $nightly_list_var; do
     echo $count
 done
 
-for i in $nightlystatus; do
+for i in "${nightlystatus[@]}" 
+do
+    echo $i
     if [ $i = "Passed" ]
```

Comment:
I don't think this is still right, you need to check here if the global is ever false and do not set it to pass if it ever becomes false.